README.md

```diff
@@ -13,52 +14,48 @@ Another reason to choose mmproxy-rs may be if you want to avoid interference fro
 ## Features
 
 - [x] TCP - Accepts PROXY Protocol enabled requests from [Nginx](https://docs.nginx.com/nginx/admin-guide/load-balancer/using-proxy-protocol/#proxy-protocol-for-a-tcp-connection-to-an-upstream), [HAProxy](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)
-- [x] UDP - Accepts PROXY Protocol enabled requests from [udppp](https://github.com/b23r0/udppp)
+- [x] UDP - Accepts PROXY Protocol enabled requests from [udppp](https://github.com/b23r0/udppp), [Cloudflare Spectrum](https://www.cloudflare.com/products/cloudflare-spectrum/)
 - [x] No Garbage Collection pauses
 
-## Usage
+## Installation
 
+Install Rust with [rustup](https://rustup.rs/) if you haven't already.
+
+```sh
+$ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+$ cargo --version
 ```
+
+### Installing with Cargo
+
+```sh
+# install from the main branch
+$ cargo install --git https://github.com/saiko-tech/mmproxy-rs
```

Comment:
should be `    cargo install --git https://github.com/saiko-tech/mmproxy-rs`

e.g.

From crates.io:

    cargo install mmproxy

From git:

    cargo install --git https://github.com/saiko-tech/mmproxy-rs